src/payment/data/service.js

```diff
@@ -33,32 +34,18 @@ export function configureApiService(newConfig, newApiClient) {
 }
 
 export function transformResults(data) {
-  const transformedResults = {
-    basketId: data.basket_id,
-    currency: data.currency,
-    isFreeBasket: data.is_free_basket,
-    showCouponForm: data.show_coupon_form,
-    orderTotal: Number.parseInt(data.order_total, 10),
-    summaryDiscounts: data.summary_discounts !== null ?
-      Number.parseInt(data.summary_discounts, 10) : null,
-    summaryPrice: Number.parseInt(data.summary_price, 10),
-    products: data.products.map(({
-      image_url: imageUrl,
-      title,
-      certificate_type: certificateType,
-      sku,
-      product_type: productType,
-    }) => ({
-      imageUrl, title, certificateType, sku, productType,
-    })),
-    coupons: data.coupons,
-    offers: data.offers ?
-      data.offers
-        .filter(({ provider }) => provider !== null)
-        .map(({ benefit_value: benefitValue, provider }) => ({ benefitValue, provider }))
-      : null,
-  };
-  return transformedResults;
+  const results = camelCaseObject(data);
```

Comment:
I updated this method because - as written - it was weeding out anything we hadn't explicitly added.  Like messages.  It took me a while to figure out what was going on.  We wrote camelCaseObject a while ago to do most of what this function was doing.  The exceptions/things that needed further processing are below.